<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatingRoom</class>
 <widget class="QDialog" name="chatingRoom">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>채팅방</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTextBrowser" name="allChat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>431</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="placeholderText">
    <string>채팅내용</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>참여자 수 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="memberCount">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInput">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="placeholderText">
    <string>채팅을 입력하세요,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInput">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>계정명 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>557</x>
     <y>27</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>계정이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>명</string>
   </property>
  </widget>
  <widget class="QListWidget" name="members">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>131</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>101</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>접속 계정</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
